pyproject.toml

```diff
@@ -28,18 +28,18 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.10.7"
-unbabel-comet = {git = "https://github.com/Unbabel/COMET.git", rev = "02d1bd2e01fa32f3c0d6266e803ec01f09cf9c8f"}
+python = ">=3.9,<3.9.7 || >3.9.7,<3.10.7"
+unbabel-comet = {git = "https://github.com/Unbabel/COMET.git"}
```

Comment:
Maybe let's set it to a specific commit, so it doesn't break when they'll change sth again.